spikingjelly/activation_based/functional/misc.py

```
@@ -13,9 +15,258 @@
+def pack_conv_bn_train_modules(net: nn.Module) -> fx.GraphModule:
+    """
+    **API Language:**
+    :ref:`中文 <pack_conv_bn_train_modules-cn>` | :ref:`English <pack_conv_bn_train_modules-en>`
+
+    ----
+
+    .. _pack_conv_bn_train_modules-cn:
+
+    * **中文**
+
+    将训练模式下模型中的相邻 ``Conv*`` 与 ``BatchNorm*`` 模块打包为单个 wrapper，
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)